File: Assignment_14/program2.c
// Accept number of rows and number of columns from user and display below pattern.

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Accept number of rows and number of columns from user and display below pattern.
// Autor       :  Ankita Anil Patil
// Date        :  22/05/2025
///////////////////////////////////////////////////////////////////////////////

/* Input : iRow = 4   iCol = 5
   Output : 2 4 6 8 10
            1 3 5 7 9
            2 4 6 8 10
            1 3 5 7 9
*/

#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0, j = 0, k = 0 ;

    for(i = 1; i <= iRow ; i++)
    {
        if((i % 2) == 0)
        {
            k = 1;
        }
        else
        {
            k = 2;
        }
        for(j = 1; j <= iCol && k <= 10; j++, k += 2)
        {
           printf("%d\t",k); 
           
        }

        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows  ");
    scanf("%d",&iValue1);

    printf("Enter number of columns ");
    scanf("%d",&iValue2);


    Pattern(iValue1,iValue2);
    return 0;
}

File: Assignment_50/program2.java
// Write a java program which accept number of rows and number of columns from user and display below pattern.
/*
    Input : iRow = 4     iCol = 4
    
    Output: * * * #
            * * # @
            * # @ @
            # @ @ @ 
 */ 

   
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.java
// Description :  Accept number of rows and number of columns from user
//                and display below pattern.            
// Autor       :  Ankita Anil Patil
// Date        :  19/08/2025
///////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Pattern 
{ 
    int Arr[][];
   
    public Pattern(int iRow, int iCol) 
    { 
       Arr = new int [iRow][iCol];     
    } 

    public void  DisplayPattern()
    {
        int i = 0, j = 0;
        
        for(i = 0; i < Arr.length; i++)
        {
            for(j = Arr.length -1; j >= 0; j--)
            {
                if(i == j)
                {
                    System.out.print("#" +"\t");
                }
                else if(i < j)
                {
                    System.out.print("*" +"\t");
                }
                else 
                {
                    System.out.print("@" +"\t");
                }
                
            }

            System.out.println();
        }

    }
}

class program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows : ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns : ");
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern(iValue1,iValue2);

        pobj.DisplayPattern();

    }
}